misk/web/tabs/config/src/actions/types.ts

```diff
@@ -0,0 +1,17 @@
+import { createActionTypes } from "./utils"
+
+export interface IActionType {
+  [base:string]: string
+}
+
+// export const CONFIG = createActionTypes("CONFIG", [
+//   "FAILURE",
+//   "GET_All",
+//   "SUCCESS"
+// ])
+
+export enum CONFIG {
+  FAILURE = "CONFIG_FAILURE",
+  GET_ALL = "CONFIG_GET_ALL",
+  SUCCESS = "CONFIG_SUCCESS"
```

Comment:
I love when variable name lengths all match up

Reply:
There is still beauty in this crazy world.